### PR TITLE
Mobile friendly Alert & Icon

### PR DESCRIPTION
**Pull request by xdrdak**
## Description

Both will now resize appropriately in their mobile breakpoints. Mostly just adjusting the size of fonts

## How to test?

- Checkout branch, run `yarn dev`
- Open [Storybook](http://localhost:6006)
- Or check the deploy preview on Netlify (link available in comments)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md) guide
- [x] I have prepared [CHANGELOGs](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md#git-workflow) for release
- [x] I have tested my changes on [supported browsers](https://browserl.ist/?q=%3E0.25%25%2C+not+op_mini+all%2C+not+ie+%3C%3D+11)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] When approved, I have run [visual regression tests](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md#running-visual-regression-tests), got it approved and [posted a link](https://percy.io/Lightspeed/flame)

### CODE REVIEW

File: packages/flame/src/Alert/Alert.tsx [hunk 1]
```diff
@@ -90,11 +94,19 @@ const Alert: React.FunctionComponent<AlertProps & SpaceProps> = ({
...
         <Box flex="1">
           {title && (
-            <Text color="textHeading" fontWeight="bold" fontSize="text" mt={0} mr={0} mb={1} ml={0}>
+            <Text
+              color="textHeading"
+              fontWeight="bold"
+              fontSize={['text-l', 'text']}
+              mt={0}
+              mr={0}
+              mb={1}
```
Comment (alexislightspeed) [inline packages/flame/src/Alert/Alert.tsx]: Design needs to go back to the drawing board on this one, they put a 3px space. It either needs to be `space.0` or `space.1`, and align with the visuals.

**Comment by alexislightspeed** [comment]
Not the cleanest way of vertically aligning the Icon with the Title:
`<Flex alignItems="center" pr={2} pt="2px">`

**Comment by xdrdak** [comment]
Risky to align the icon with title that way.

Might have to change the markup a bit to have flex do its thing properly

File: packages/flame/src/Alert/Alert.tsx [hunk 2]
```diff
@@ -11,40 +11,45 @@ const alertStyles = variant({
...
 `;
 
 AlertWrapper.defaultProps = {
   type: 'info',
+  px: 3,
+  py: [3, 2],
 };
 
-const CloseButton = styled('button')`
+const CloseButton = styled('button')<LayoutProps>`
```
Comment (alexislightspeed) [inline packages/flame/src/Alert/Alert.tsx]: The Button & Icon should both be vertically centered with the Title.

File: packages/flame/src/Alert/Alert.tsx [hunk 3]
```diff
@@ -90,11 +95,19 @@ const Alert: React.FunctionComponent<AlertProps & SpaceProps> = ({
         )}
         <Box flex="1">
           {title && (
-            <Text color="textHeading" fontWeight="bold" fontSize="text" mt={0} mr={0} mb={1} ml={0}>
+            <Text
+              color="textHeading"
+              fontWeight="bold"
+              fontSize={['text-l', 'text']}
```
Comment (alexislightspeed) [inline packages/flame/src/Alert/Alert.tsx]: `line-height` isn't following along, for some reason. Both `text` and `text-l` should have a `line-height` of 24.

File: packages/flame/src/Alert/Alert.tsx [hunk 4]
```diff
@@ -90,11 +95,19 @@ const Alert: React.FunctionComponent<AlertProps & SpaceProps> = ({
         )}
         <Box flex="1">
           {title && (
-            <Text color="textHeading" fontWeight="bold" fontSize="text" mt={0} mr={0} mb={1} ml={0}>
+            <Text
```
Comment (alexislightspeed) [inline packages/flame/src/Alert/Alert.tsx]: line 92: aligning the icon vertically like this is not sustainable. Can we `flex` it?

**Pull request merged**